addons/binding/org.openhab.binding.seneye/README.md

```diff
@@ -4,54 +4,58 @@ This binding integrates the [Seneye aquarium monitoring system](https://www.sene
 
 ## Introduction
 
-Understanding what is happening inside your aquarium is vital to ensuring that the aquatic life remains healthy. This revolutionary water monitoring device allows you to continuously track the changes in the water parameters, alerting you to the problems before they affect the fish. Protect your fish with a seneye monitor.
+The seneye monitor monitors what is happening inside your aquarium to ensure that the aquatic life remains healthy.
+The monitor allows you to continuously track the changes in the water parameters, alerting you to the problems before they affect the fish. 
 
 At least one Seneye monitor is required ([Home / pond or reef](https://www.seneye.com/devices/compare)) and the measure results must be synced to the seneye cloud by using a seneye web server (see [shop](https://www.seneye.com/store), there is one for wifi and one for a wired network)
-Each monitor is represented by 1 openhab 2 seneye thing.
+Each monitor is represented by 1 openhab seneye thing.
 
 ## Supported Things
 
-This binding provides 1 thing type : 'seneye'. You can have multiple seneye devices in your home, just make sure that your aquariumname is properly set for each seneye thing.
+This binding provides one thing type : 'seneye'. You can have multiple seneye devices in your home, just make sure that your aquariumname is properly set for each seneye thing.
 
 ## Discovery
 
 Discovery is not supported, the seneye monitor must be configured manually
 
 ## Thing Configuration
 
-### Configuration in PaperUi
+### Configuration in PaperUI
 
-Following settings must be configured in order to make your seneye binding work :
+The following settings must be configured in order to make your seneye binding work:
 
-Aquariumname : the name of the aquarium, as specified in [seneye.me] (https://www.seneye.me/). Usefull to distinguish multiple seneye installations.
-Username : your login name for [seneye.me] (https://www.seneye.me/)
-Password : your password for [seneye.me] (https://www.seneye.me/)
-Polling time : How often (in minutes) does the seneye needs to be checked ?
+| Setting              |                                                                                 |
+|----------------------|---------------------------------------------------------------------------------|
+| Aquariumname         | The name of the aquarium, as specified in [seneye.me] (https://www.seneye.me/). |
```

Comment:
There is an underscore missing and you might also want to use lower case letters.

Reply:
ok